src/bidiMapper/rollup.config.js

```diff
@@ -40,5 +41,10 @@ export default {
     }),
     nodeResolve(),
     commonjs(),
+    terser({
+      format: {
+        comments: /copyright/i,
```

Comment:
Does it keep all the copyrights comments? If so, it seems an overkill, and we keep only 1 at the beginning of the compiled file.

Reply:
Yes, all comments. If it's desired to keep only one, then we could remove this and just add `@license` in the jsdoc of the sole comment we want to preserve.

Reply:
I'm not sure if we want to keep the license in the compiled version at all, but even if we do, let's don't duplicate it.

Reply:
OK. Follow-up: https://github.com/GoogleChromeLabs/chromium-bidi/pull/223